README.md

```diff
@@ -21,52 +21,58 @@ Java Rest Api Built with Spring Framework on a postgres database deployed to Her
 
 ## Endpoints
 
+All Endpoints that return lists take pageable query paramaters `page, size, & sort`
+
 #### User Routes
 
 | Method | Endpoint                | Access Control      | Description                                        |
 | ------ | ----------------------- | ------------------- | -------------------------------------------------- |
 | GET    | `/users/users`                | Admin               | Returns paginated list of all users                |
 | GET    | `/users/users/:userId`        | Admin               | Returns info for a single user.                    |
+| GET    | `/users/usertype`        | Admin               | Returns roles of current user.                    |
```

Comment:
User Type is actually accessible to all, otherwise our 401 middleware would logout any non-admin user immediately